tools/taskfiles/taskfile_code_quality.yml

```diff
@@ -41,6 +41,7 @@ tasks:
         sh: suffix="|| true"; if [ "{{.STRICT|default false}}" == "true" ]; then suffix=""; fi; echo $suffix
     cmds:
       - |
+        set +ex
```

Comment:
Why this is needed? Won't removing `silent: true` do the trick?

Reply:
Sadly it's not helping with that issue. 
Silent mode disables the echoing of commands